test_apps/test_app/app/contracts/ENSRegistry.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity ^0.4.18;
+
+import './ENS.sol';
+
+/**
+ * The ENS registry contract.
+ */
+contract ENSRegistry is ENS {
```

Comment:
I am not sure to understand completely the test_app yet.
But why do you copy the contract here, by configuring ENS inside the test app, the contract should be deployed no?